tests/acceptance/standard/test_group_avatar.py

```diff
@@ -0,0 +1,75 @@
+import os
+import pytest
+from tests.acceptance import (
+    run_gitlabform,
+)
+
+
+class TestGroupAvatar:
+    def setup_method(self):
+        # Use gitlabform logo
+        self.project_root = os.path.abspath(os.path.join(os.path.dirname(__file__), "../../.."))
+        self.test_image_path = os.path.join(self.project_root, "docs/images/gitlabform-logo.png")
```

Comment:
The documentation for this feature suggests both absolute and relative path in the config is supported. Looks like the test cases are only validating absolute path. Can we add a validation for relative path too?

Reply:
Thanks for the updates.